paywall/src/__tests__/services/walletService.test.js

```diff
@@ -16,6 +16,7 @@ import {
 import { POLLING_INTERVAL } from '../../constants'
 
 jest.mock('../../utils/promises')
+jest.unmock('web3')
```

Comment:
I'm a bit curious about these, is this because `web3` is mocked elsewhere and jest stores that between tests?

LG overall!

Reply:
if you have 3 minutes, try checking out the branch and removing that line. The difference is that I made a global mock for the web3 node module, and we enabled automocking, so it is instantly mocked in every test. I spent HOURS trying to find a way to mock the damn thing only in 1 test, and jest does not seem to make it possible. If there is a way, I will do that instead in a heartbeat because this way sucks